torch/csrc/jit/operator_upgraders/version_map.cpp

```diff
@@ -16,7 +16,15 @@ static bool isVersionMapSorted = false;
 // Note for developers: The list of upgraders should be SORTED
 // by the version number where the upgrader is registered.
 static std::unordered_map<std::string, std::vector<UpgraderEntry>> operatorVersionMap(
-    {{"aten::div.Tensor",
+    {{"aten::linspace",
+      {{8,
```

Comment:
OK now I see why the number is 8 and this is linspace 0 through 7, but why isn't it linspace 0 through 6 and version 7?

Reply:
Explained here: (https://github.com/pytorch/pytorch/pull/71486#discussion_r791207895)